packages/espree/test-d/index.test-d.ts

```diff
@@ -0,0 +1,60 @@
+import { expectType, expectAssignable } from 'tsd';
+
+import * as espree from "../dist/espree.cjs";
+import * as acorn from "acorn";
+import type { Options, EspreeTokens } from "../dist/espree.cjs";
```

Comment:
```suggestion
import * as espree from "../dist/espree.cjs";
import * as acorn from "acorn";
```

Maybe, we could also simplify the import by using `espree.Options` and `espree.EspreeTokens`, similar to using `acorn.Node`, instead of destructuring them from the same import source (`../dist/espree.cjs`). What do you think?